infrastructure/template.yaml

```
@@ -1624,6 +1628,11 @@ Resources:
             - auditEventQueueName: !ImportValue "AuditEventQueueName"
           Id: AuditEventQueue
           InputPath: "$.detail"
+          DeadLetterConfig: !If
+              - IsDevLikeEnvironment
+              - !Ref AWS::NoValue
+              - Arn: !ImportValue
+                  Fn::Sub: "${TXMAStackName}-AuditEventDLQArn"
```

```suggestion
              - Arn:
                  Fn::ImportValue: !Sub: ${TxmaStackName}-AuditEventDLQArn
```
Again just for constitency